---
source: src/validation/tests/generic_validation_tests.rs
expression: make_readable(&diagnostics)
---
SyntaxError { message: "Invalid type nature for generic argument. REAL is no Chars.", range: [SourceRange { range: 431..439 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. UDINT is no Chars.", range: [SourceRange { range: 441..453 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. DINT is no Chars.", range: [SourceRange { range: 455..465 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. TIME is no Chars.", range: [SourceRange { range: 467..475 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. BYTE is no Chars.", range: [SourceRange { range: 477..485 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. DATE is no Chars.", range: [SourceRange { range: 506..514 }], err_no: type__invalid_nature }

